keyboards/kelowna/rgb64/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = STM32F303
+BOARD = QMK_PROTON_C
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```

Comment:
1. For STM32-based boards you really should use either `WS2812_DRIVER = pwm` or `WS2812_DRIVER = spi` instead of the default `bitbang` driver; however, those drivers may require using some specific pins and need some configuration both in `config.h` and `mcuconf.h`. I'm not sure whether the `#define RGB_DI_PIN B1` that you have in `config.h` at the moment is correct (other pin definitions look wrong for STM32F303, so maybe you have a wrong version of the file in this PR), but the `B1` pin should be usable with `WS2812_DRIVER = pwm` on STM32F303 with some additional setup (but it's not 5V tolerant, therefore it would probably need some logic level shifting circuit).
2. `RGBLIGHT_ENABLE = yes` is not intended for use with per-key RGB LEDs (it's mainly intended for RGB underglow). The proper way to support per-key RGB LEDs is `RGB_MATRIX_ENABLE = yes`, but it requires more configuration to describe the RGB matrix layout.